src/Template/Common/ComposerJsonTemplate.php

```diff
@@ -30,9 +30,9 @@ public function __toString(): string
                     "nesbot/carbon": "^2.0",
                     "emulgeator/enum": "^1.0",
                     "emulgeator/array-to-class-mapper": "^0.1|^1.0",
-                    "psr/http-message": "^1.0",
+                    "psr/http-message": "^2.0",
```

Comment:
If there are no compatibility issues for our use case (check what changed between 1.0 and 2.0), should support both 1.0 and 2.0